[chartdir] build failure

**Host Environment**
- OS: [Windows10]
- Visual studio 2022

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install chartdir`

**Failure logs** 
C:\src\vcpkg>vcpkg install chartdir
Computing installation plan...
The following packages will be built and installed:
    chartdir[core]:x86-windows -> 7.0.0#2
Detecting compiler hash for triplet x86-windows...
Restored 0 packages from C:\Users\jingm\AppData\Local\vcpkg\archives in 116.9 us. Use --debug to see more details.
Starting package 1/1: chartdir:x86-windows
Building package chartdir[core]:x86-windows...
-- Downloading https://www.advsofteng.net/chartdir_cpp_win.zip -> chartdir_cpp_win-7.0.0.zip...
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'manifests' = off
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Downloading https://www.advsofteng.net/chartdir_cpp_win.zip
Error: Failed to download from mirror set:
File does not have the expected hash:
             url : [ https://www.advsofteng.net/chartdir_cpp_win.zip ]
       File path : [ C:\src\vcpkg\downloads\chartdir_cpp_win-7.0.0.zip.10324.part ]
   Expected hash : [ 38d9dae641c0341ccee4709138afd37ad4718c34def70a0dc569956bf9c3488d0d66072f604dca4663dc80bd09446a2ba27ef3806fc3b87dda6aaa5453a7316f ]
     Actual hash : [ e5b5d387cff693a7f5ee98c2d2df75f421129b006e4324ae30ace0cbaac58867f048868ddfacdb3224c7165c8f27219c4273f3c778be3330d39ef95260d4186b ]


[DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(709):
[DEBUG] Time in subprocesses: 0 us
[DEBUG] Time in parsing JSON: 2 us
[DEBUG] Time in JSON reader: 0 us
[DEBUG] Time in filesystem: 481 us
[DEBUG] Time in loading ports: 0 us
[DEBUG] Exiting after 6.051 s (6044657 us)

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:84 (message):

      Failed to download file with error: 1
      If you use a proxy, please check your proxy setting. Possible causes are:

      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.

      2. You are using Fiddler. Currently a bug (https://github.com/microsoft/vcpkg/issues/17752)
         will set HTTPS_PROXY to `https://fiddler_address:port` which lead to problem 1 above.
         Workaround is open Windows 10 Settings App, and search for Proxy Configuration page,
         Change `http=address:port;https=address:port` to `address`, and fill the port number.

      3. Your proxy's remote server is out of service.

      In future vcpkg releases, if you are using Windows, you no longer need to set
      HTTP(S)_PROXY environment variables. Vcpkg will simply apply Windows IE Proxy
      Settings set by your proxy software. See (https://github.com/microsoft/vcpkg-tool/pull/49)
      and (https://github.com/microsoft/vcpkg-tool/pull/77)

      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:309 (z_vcpkg_download_distfile_show_proxy_and_fail)
  ports/chartdir/portfile.cmake:9 (vcpkg_download_distfile)
  scripts/ports.cmake:145 (include)


Error: Building package chartdir:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `git pull` and `.\vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+chartdir
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[chartdir]+Build+error
including:
  package: chartdir[core]:x86-windows -> 7.0.0#2
    vcpkg-tool version: 2022-03-09-1affd32f93b299d5a907816c328ca3ededb73a7e
    vcpkg-scripts version: a3df69692 2022-03-18 (2 days ago)

Additionally, attach any relevant sections from the log files above.

**Additional context**
I met this on both of my laptop and desktop.  I am working on a optical fiber project which needs package wxwidget, matio, chartdir and wxchartdir, among all the packages involved, chartdir cannot be built correctly.

Does someone know what is the problem and the solution?
Many thanks.

--- a/ports/chartdir/portfile.cmake
+++ b/ports/chartdir/portfile.cmake
@@ -9,7 +9,7 @@ elseif(VCPKG_TARGET_IS_WINDOWS)
     vcpkg_download_distfile(ARCHIVE_FILE
         URLS "https://www.advsofteng.net/chartdir_cpp_win.zip"
         FILENAME "chartdir_cpp_win-7.0.0.zip"
-        SHA512 38d9dae641c0341ccee4709138afd37ad4718c34def70a0dc569956bf9c3488d0d66072f604dca4663dc80bd09446a2ba27ef3806fc3b87dda6aaa5453a7316f
+        SHA512 e5b5d387cff693a7f5ee98c2d2df75f421129b006e4324ae30ace0cbaac58867f048868ddfacdb3224c7165c8f27219c4273f3c778be3330d39ef95260d4186b
     )
 
     vcpkg_extract_source_archive_ex(

--- a/ports/chartdir/vcpkg.json
+++ b/ports/chartdir/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "chartdir",
   "version": "7.0.0",
-  "port-version": 2,
+  "port-version": 3,
   "description": "ChartDirector is a powerful chart component for creating professional looking charts for web and windows applications.",
   "homepage": "https://www.advsofteng.com/"
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1338,7 +1338,7 @@
     },
     "chartdir": {
       "baseline": "7.0.0",
-      "port-version": 2
+      "port-version": 3
     },
     "check": {
       "baseline": "0.15.2",

--- a/versions/c-/chartdir.json
+++ b/versions/c-/chartdir.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "2b4bad1eb46439cbd63295d8120f03966c52d162",
+      "version": "7.0.0",
+      "port-version": 3
+    },
+    {
       "git-tree": "fe0cf0c045f1092f18faa48f70b6b23d66e6298e",
       "version": "7.0.0",
       "port-version": 2

